hawkeye-maven-plugin/src/test/java/io/korandoru/hawkeye/RemoveMojoTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Korandoru Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.korandoru.hawkeye;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RemoveMojoTest {
+
+    RemoveMojo removeMojo;
+    File tempFile;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        tempFile = File.createTempFile("test", ".yaml", new File("src/test/resources/test_remove"));
```

Comment:
There are no files added by Git in the dir `test_remove` so the dir doesn't exist so the test failed during GH workflows.

Reply:
@tisonkun I think it's good to go for CI now.